test/integration/etcd/server.go

```diff
@@ -67,17 +66,6 @@ AwEHoUQDQgAEH6cuzP8XuD5wal6wf9M6xDljTOPLX2i8uIp/C/ASqiIGUeeKQtX0
 /IR3qCXyThP/dbCiHrF3v1cuhBOHY8CLVg==
 -----END EC PRIVATE KEY-----`
 
-func registerEffectiveEmulationVersion(t *testing.T) {
```

Comment:
note to self/reviewers: This is a bug discovered in https://github.com/kubernetes/kubernetes/pull/128279#discussion_r1908013803. Emulation version could have been changed by the previous test and must be reset so this needs to run for all tests rather than selectively calling the function.